### DISCUSSION TITLE
help to create bash script to create the postgresql container on 1 panel

### DISCUSSION THREAD

**Discussion by tpchagas** [1Panel-dev/1Panel#7520] [General]
Hello
I am automating the installation of 1panel using a bash script where I already install other tools

The installation of 1panel is happening normally through my script.

What I would like to know is if there is documentation or examples so that

through the shell, using a script, I can install PostgreSQL from the AppStore, in the same way that the app is created in the 1panel frontend app store.

And in the same way, I would like to do it through the script, I would like to create a database in the PostgreSQL Docker container created by the App Store.

Because I use the 1panel cronjob and the backup account to send my database to OneDrive

**Comment by wanghe-fit2cloud** [comment]
Thanks for your feedback! 
Currently, we do not provide a solution for installing applications via Shell scripts.
If you need to create a database, you can use the docker exec command to execute the create database statement.
